p2p/sharding.py

```diff
@@ -148,35 +233,15 @@ def __init__(self, shard: Shard, peer_pool: PeerPool, token: CancelToken=None) -
 
         self.shard = shard
         self.peer_pool = peer_pool
+        self._running_peers: Set[ShardingPeer] = set()
 
-        self.incoming_collation_queue: asyncio.Queue[Collation] = asyncio.Queue()
-
-        self.collations_received_event = asyncio.Event()
+        self.collation_hashes_at_peer: Dict[ShardingPeer, Set[Hash32]] = defaultdict(set)
 
         self.start_time = time.time()
 
     async def _run(self) -> None:
```

Comment:
Just to be sure, we will to add the `_sync_complete` event condition after we have SMC info, right?

Reply:
Yep, we can add all kinds of events when we know what we need. For now the syncer just runs continuously and doesn't know when it's complete, so there's no event for it.